### ISSUE TITLE
RealTabFormer Preprocessing Inconsistent Column Size

### ISSUE THREAD

**Issue by R-N** [R-N/ml-utility-loss#4]
RealTabFormer preprocessing truncates the trailing zero even after the precision. This makes the column size inconsistent so I removed it. Furthermore, it doesn't take into account the possibility of negative sign for integer that doesn't exist in the training set. Maybe it was intentional? But anyway, it makes the column length of the preprocessed input inconsistent when the test set has it, and the number doesn't align well as a string. So I make it add another space for the negative sign when the training set doesn't have it. This seems to be automatic for floats though.
